plugin/converter/template.go

```diff
@@ -36,7 +36,7 @@ import (
 
 var (
 	// templateFileRE regex to verifying kind is template.
-	templateFileRE              = regexp.MustCompile("^kind:\\s+template+\\n")
+	templateFileRE              = regexp.MustCompilePOSIX("^kind:[[:space:]]+template[[:space:]]?\\+?")
```

Comment:
are you just checking for the existance for a single line ie
```kind: template``` where there can be any number of spaces between the `kind:` and `template`. Why are you matching characters after the e in template ? should you not be checking for a newline ?

Reply:
Not fully understanding you here TP...

Reply:
I've updated to remove the \\+?, might just have been a typo when he sent it across.